frontend/src/features/backoffice/edit_regulation/EditRegulation.js

```diff
@@ -100,6 +101,12 @@ const EditRegulation = ({ title, isEdition }) => {
     regulatoryReferences
   } = processingRegulation
 
+  useEffect(() => {
```

Comment:
L'idée du `IDLE` était pour moi un moyen d'éviter ça.

En clair dans un formulaire de création et d'édition, `IDLE` par défaut (et donc aussi comme valeur de reset), `LOADING` lorsqu'on charge et `READY` (ou `LOADED`) quand c'est chargé : https://redux.js.org/tutorials/essentials/part-5-async-logic#loading-state-for-requests.

Dans ce fix, ça serait du coup `IDLE` ou `READY` qui laisseraient les reducers d'update s'exécuter et non seulement `READY`. Non ?